assets/src/scripts/components/MoreInfoModal.tsx

```diff
@@ -118,9 +119,11 @@ function MoreInfoModal({
             data.synonyms[code].filter((synonym: string) => synonym !== term) ||
               [],
           );
+          setReferences(data.references[code] || []);
```

Comment:
I can't do the suggestion for some reason in the UI, but you'll need to make sure that we're checking for the right data when only synonyms or references are being returned, otherwise the filter may fail:

```jsx
setSynonyms(
  data.synonyms?.[code]?.filter(
    (synonym: string) => synonym !== term,
  ) || [],
);
setReferences(data.references?.[code] || []);
```

Optional chaining the values here will do that.

Reply:
It's annoying isn't it?

I've integrated those changes manually.